dialogflow/detect_intent_texts.rb

```diff
@@ -0,0 +1,64 @@
+# Copyright 2018 Google, Inc
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+require "securerandom"
+
+def detect_intent_texts project_id:, session_id:, texts:, language_code:
+  # [START dialogflow_detect_intent_texts]
+  # project_id = "Your Google Cloud project ID"
+  # session_id = "mysession"
+  # language_code = "en-US"
```

Comment:
Missing `texts` variable. So I can't see what kind of input is expected.

Reply:
Also, align =

Reply:
done except for = alignment.